tests/mamba/test_selective_state_update_mtp.py

```
@@ -256,41 +229,56 @@ def inputs(
+        dstate,
+        cache_steps,
+        state_dtype,
+        weight_dtype,
+        use_out_tensor,
+    ):
         """Test that kernel output matches reference but state is not updated."""
-        y_ref, state_ref = reference_output
+        inputs = self.make_inputs(
+            batch, nheads, dim, dstate, cache_steps, state_dtype, weight_dtype
+        )
+        y_ref, state_ref = self.make_reference_output(inputs)
```

Ruff (0.15.1)

[warning] Unpacked variable `state_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)